modules/auxiliary/scanner/http/ntlm_info_enumeration.rb

```diff
@@ -105,7 +105,7 @@ def check_url(test_uri)
     return if res.nil?
 
     vprint_status("Status: #{res.code}")
-    if res and res.code == 401 and res['WWW-Authenticate'].match(/^NTLM/i)
+    if res and res.code == 401 and res['WWW-Authenticate'] and res['WWW-Authenticate'].match(/^NTLM/i)
```

Comment:
Use && for logical and. Although the module was using the "and" before your PR, still the new changes should follow the convention.